core/src/main/scala-3/spire/syntax/Ops.scala

```diff
@@ -0,0 +1,17 @@
+package spire
+package syntax
+
+import spire.algebra._
+import spire.algebra.partial._
+
+final class GroupoidCommonOps[A](lhs: A)(implicit ev: Groupoid[A]):
+  def inverse(): A = ev.inverse(lhs)
+  def isId(implicit ev1: Eq[A]): Boolean = ev.isId(lhs)(ev1)
+
+final class LeftModuleOps[V](x: V):
```

Comment:
Is it worth making some of these classes `extends AnyVal`?

Reply:
Yes, we should do that